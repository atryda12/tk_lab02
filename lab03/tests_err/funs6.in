
int a = 0, b = 0, c = 0;

int pow(int n, int m) {
        int i=1;
        int pow = 1;
        while(i<=m) {
            res = pow*n;
            i = i+1;
        }
        return pow;
}

print pow(2,10);

