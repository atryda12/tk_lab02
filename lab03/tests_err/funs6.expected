
Error: Function identifier 'pow' used as a variable: line 6
Error: Variable 'res' undefined in current scope: line 8
